Smartling.API/Dashboard/DashboardApiClient.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Smartling.Api.Authentication;
+using Smartling.Api.Model;
+using System.Security.Authentication;
+
+namespace Smartling.Api.Dashboard
+{
+    public class DashboardApiClient : ApiClientBase
+    {
+        private readonly IAuthenticationStrategy _auth;
+        private const string DashboardUrl = "/translator-dashboard-api/v2/jobs";
+
+        public DashboardApiClient(IAuthenticationStrategy auth)
+        {
+            _auth = auth;
+        }
+
+        public virtual DashboardData GetDashboardData()
+        {
+            var uriBuilder = GetRequestStringBuilder(DashboardUrl);
+            var request = PrepareGetRequest(uriBuilder.ToString(), _auth.GetToken());
+            JObject response;
+
+            try
+            {
+                response = JObject.Parse(GetResponse(request));
```

Comment:
Why not to use more high-level method `ExecuteGetRequest`?

Reply:
Why is this method more higher-level if it uses GetResponse()? Also, it marks "Retry the request in case of authentication error" and all other code in this situation uses GetResponse().